### PR TITLE
SBERDOMA-579 Replace side menu shadow with border

### PR DESCRIPTION
**Pull request by MrFoxPro**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts [hunk 1]
```diff
@@ -60,7 +60,9 @@ export const sideMenuDesktopCss = css`
   top: 0;
   left: 0;
   z-index: 10;
-  ${shadow}
+  border-width: 0 1px 0 0;
+  border-color: ${colors.lightGrey[5]};
```
Comment (AntonAL) [inline apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:64]: Color should be `#f0f0f0`, now it has value `#d9d9d9`

Comment (MrFoxPro) [reply apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:64]: @AntonAL here it is color from figma
изображение

Comment (AntonAL) [reply apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:64]: Hmm, it seems, that color of a border is inconsistent between different pages
[Here](https://www.figma.com/file/P827yyueCLRT6o4gyNtzY6/SberDoma-(ANT)?node-id=3905%3A64386), for example:
image

Comment (AntonAL) [reply apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:64]: Waiting for answer from Maks

Comment (MrFoxPro) [reply apps/condo/domains/common/components/containers/BaseLayout/components/styles.ts:64]: He said to use grey-5 color.

**Pull request merged**